### ISSUE TITLE
docker-compose

### ISSUE THREAD

**Issue by gerisse** [wernight/docker-codiad#5]
I try to run this docker image with traefik, but without succes
i think the problem is cause by the esposed port in your container, 8080, that is also the port used by traefik
is it possible to change the port 8080 ?
thanks
ps :my traefik container works always with ssl ecrynption (Lets'encrypt certificat)

**Comment by wernight** [comment]
Yes of course. It's standard docker. Same as for any container via -p

**Comment by saytosid** [comment]
You can map it to a different port using:
```
ports:
  - 1234:8080
```

Or better, don't bind a port from this image to the host but just expose it and tell traefik to redirecto to the exposed port. This way is more secure as now external connections cannot come to this container directrly
